logger.py

```diff
@@ -1,6 +1,43 @@
 import logging
+import logging.config
+import yaml
 
-import sys
+APP_ROOT = 'bt-mqtt-gw'
```

Comment:
So you suggest to use the folder name where `gateway.py` resides in?

Reply:
Yes, I image someone can clone it to whatever directory wants, or to some common dir etc.

Reply:
In the end, this is just a marker for logging purposes. It appears as prefix in the name of a concrete logger instance when running in debug mode. It doesn't serve any other purpose. I just liked it more to have the project name in front instead of `gateway` (resulting from `gateway.py`). Example:
```
2019-03-01 21:22:14,859 DEBUG bt-mqtt-gw.workers.mithermometer mithermometer.py:20:_setup - Adding mithermometer device 'wohnen' (4C:65:A8:D6:01:18)
2019-03-01 21:22:10,621 DEBUG bt-mqtt-gw.workers_manager workers_manager.py:54:register_workers - Added thermostat worker with 300 seconds interval
2019-03-01 21:22:14,888 INFO bt-mqtt-gw.workers.thermostat thermostat.py:74:status_update - Updating 5 thermostat devices
```
But I could implement your suggestion nonetheless. :)

Reply:
I see, nvm them, sorry for confusion :D